marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/matching.cy.tsx

```diff
@@ -629,25 +602,21 @@ describe("Matching", () => {
     cy.findByText("A title for this list is required.");
     cy.findByText("Values to ignore in this list are required.");
 
-    // Don't allow values to ignore with special characters
```

Comment:
Would it be good to add a cy.log?

Reply:
It is already clear here:
![image](https://user-images.githubusercontent.com/126084579/235224839-ba306dfe-d0e8-4396-ade5-3cd5e825e856.png)

If you think that the message is not clear enough, that means, for clients probably is not clear enough, so we will need to update the error message